_unit-test/create-docker-volumes-test.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+source "$(dirname $0)/_test_setup.sh"
+
+get_volumes() {
+  # If grep returns no strings, we still want to return without error
+  docker volume ls --quiet | { grep '^sentry-.*' || true; } | sort
+}
+
+# Maybe they exist prior, maybe they don't. Script is idempotent.
+
+expected_volumes="sentry-clickhouse
+sentry-data
+sentry-kafka
+sentry-postgres
+sentry-redis
+sentry-symbolicator
+sentry-zookeeper"
+
+before=$(get_volumes)
+
+test "$before" == "" || test "$before" == "$expected_volumes"
+
+source create-docker-volumes.sh
```

Comment:
Are we in the right place for this `source` to work? Do the paths line up? Same in other files ...

Reply:
yep, the `_test_setup.sh` file does a cd into the `install` directory